### ISSUE TITLE
Get the Android build working (buildozer)

### ISSUE THREAD

**Issue by metamarcdw** [metamarcdw/nowallet#8]
**Is your feature request related to a problem? Please describe.**
Currently the buildozer spec file is not producing a functional APK, even on the Kivy Complete build VM.
https://github.com/Zen-CODE/kivybits/blob/master/KivyCompleteVM/ReadMe.txt

**Describe the solution you'd like**
Develop a working buildozer spec and start building Android releases.

**Comment by metamarcdw** [comment]
Discovered a docker image that seems more up to date than the VM, could be worth a try. AFAIK we still need Cython==0.24, which would need to be changed in the requirements file for the docker image.

https://github.com/diegodukao/docker-python3-kivy-buildozer

**Comment by xavierfiechter** [comment]
Has to install `pip install Cython==0.29.6` and `pip install pygame` to get the code build on OS X/macOS 10.14.2. Cython==0.24 is incompatible with Python 3.7+

**Comment by metamarcdw** [comment]
You were able to get it running though? We currently only test with Python 3.5 and 3.6. Did you need to tweak the buildozer.spec?

**Comment by xavierfiechter** [comment]
I was able to launch the app from command line using `make go-kivy`. I do not have Tor running at the moment on this machine so I'm stucked here for the moment. Will have a look later again.

**Comment by metamarcdw** [comment]
That's awesome! Are you thinking about giving the Android build a go? I ask because that is the purpose of this issue.

**Comment by xavierfiechter** [comment]
Yes @metamarcdw –  because from my personal point of view this is the most useful target platform for such a wallet.
See, https://twitter.com/xavierfiechter/status/1115204178257874944

**Comment by metamarcdw** [comment]
I agree. The .deb build using PyInstaller would be mainly for use on TAILS and for testing purposes.

**Comment by xavierfiechter** [comment]
@metamarcdw would you agree to integrate Tor directly? Samourai did that a few weeks ago. It's really convenient.  

Did not tested it until now, but maybe https://github.com/Marten4n6/TinyTor is an option for this.

**Comment by metamarcdw** [comment]
I most definitely would agree to that. That would be amazing!

**Comment by xavierfiechter** [comment]
Tor is up and running on this testing machine now.

Had issues with the exchange rate calls against bitcoinaverage. Could bypass this by returning a static JSON. 

Question @metamarcdw : Wouldn't it be better in the long run to ditch the fiat values? Samourai did this a few weeks back. I do not have much BTC on my stack at all but I measure in mBTC or sats and not in fiat. 
Somebody who uses a brain wallet is, I assume, not interested too much in fiat anyways.

**Comment by metamarcdw** [comment]
You bring up a valid point, and I was thinking about following Samourai's lead on this, but I'm not sure if our use case calls for it.
I'm specifically imagining a sort of international Hawala agent using Nowallet to carry BTC in and out of repressive states, in which case the exchange rates might be crucial. What are your thoughts on this type of scenario?

**Comment by xavierfiechter** [comment]
I would ditch the fiat pairs. :-)
